app/controllers/claims/support/claims_controller.rb

```diff
@@ -1,2 +1,11 @@
 class Claims::Support::ClaimsController < Claims::Support::ApplicationController
+  before_action :set_school, only: [:index]
+
+  def index; end
+
+  private
+
+  def set_school
+    @school = School.find(params[:school_id])
+  end
```

Comment:
Excuse my oar.... I'm sure there must be a reason for this, but if we're just looking at one school, isn't that a show method, not an index method?

Reply:
This is because `GET /support/schools/:school_id/claims` is routing to `support/claims#index`. I'll be adding an additional namespace to these kinds of controllers in a future PR, so the route will go to `support/claims/schools/claims#index` to signal that we want the claims belonging to a school.